packages/application-config/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://unpkg.com/@commercetools-frontend/application-config/schema.json",
+  "title": "JSON schema for Custom Application configuration files",
+  "type": "object",
+  "definitions": {
+    "envVariablePlaceholder": {
+      "type": "string",
+      "pattern": "(\\$\\{env:\\w+\\})+"
+    },
+    "cspDirective": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "uniqueItems": true
+    }
+  },
+  "properties": {
+    "id": {
+      "description": "The ID of the Custom Application (when you register the application in the Merchant Center)",
+      "type": "string"
+    },
+    "name": {
+      "description": "The name of the Custom Application",
+      "type": "string"
+    },
+    "entryPointUriPath": {
+      "description": "The unique route path of the Custom Application. This is the identifier that the Merchant Center Proxy uses to match the HTTP request and forward the request to the Custom Application. This value also needs to be used in the application client side routes. The value must be the same as the `navbarMenu.uriPath` value.",
+      "type": "string"
+    },
+    "cloudIdentifier": {
+      "description": "The cloud identifier where the Custom Application is running. This value is used to derive the Merchant Center API URL. Alternatively you can use the `mcApiUrl` property.",
+      "type": "string",
+      "oneOf": [
+        {
+          "$ref": "#/definitions/envVariablePlaceholder"
+        },
+        {
+          "enum": [
+            "gcp-au",
+            "gcp-eu",
+            "gcp-us",
+            "aws-fra",
+            "aws-ohio"
+          ]
+        }
+      ]
+    },
+    "mcApiUrl": {
+      "description": "The URL of the Merchant Center API. We usually recommend to use the `cloudIdentifier` option to avoid possible typos.",
+      "type": "string"
+    },
+    "env": {
+      "description": "This is an object of keys that represent different environments (for example `production`). The environment used depends on the environment variable MC_APP_ENV. If MC_APP_ENV isn't set then NODE_ENV will be used, if it's not set then it defaults to \"development\"",
+      "type": "object",
+      "properties": {
+        "production": {
+          "description": "Configuration for production only",
+          "type": "object",
+          "properties": {
+            "url": {
+              "description": "The URL where the Custom Application is hosted",
+              "type": "string"
+            },
+            "cdnUrl": {
+              "description": "The URL where the Custom Application static assets are hosted, like an external CDN. If the static assets are hosted alongside the Custom Application, you can omit this option and the Custom Application URL will be used instead.",
+              "type": "string"
+            }
+          },
+          "additionalProperties": false,
```

Comment:
When you see this value within the schema, it means that the related schema object does not accept any extra properties that are not defined in the schema.